source/MaterialXView/Viewer.cpp

```diff
@@ -247,7 +247,10 @@ Viewer::Viewer(const std::string& materialFilename,
     _bakeHdr(false),
     _bakeAverage(false),
     _bakeOptimize(true),
-    _bakeRequested(false)
+    _bakeRequested(false),
+    _bakeWidth(0),
+    _bakeHeight(0),
+    _bakeFilename("")
```

Comment:
This initializer can be omitted, since FilePath has a default constructor with the same behavior.